timelib_private.h

```diff
@@ -115,21 +115,21 @@
 		m = NULL;   \
 	}
 
-typedef struct ttinfo
```

Comment:
FWIW; Usual pattern for older compiler support here would be:

```
typedef struct _ttinfo {
  ...
} ttinfo;
```

Reply:
indeed, I think having a "struct _ttinfo" will be better.

BTW, the issue is with the "typedef" which is defined in the non private header, so unneeded here

Reply:
Oh, I see.  There's an opaque typedef in timelib.h as well. Your PR makes way more sense now.

#ignoreMe